### PR TITLE
add some csv details

### PR DESCRIPTION
**Pull request by mcampos-quinn**
Hey Andrew!
I was playing around with this to see if it's something that would work for us and I added some details to the output csv. Started making  a pull request as you pushed some changes, which I think I incorporated successfully into the PR.... if not, disregard this and I'll try again!

### CODE REVIEW

File: audioqc.rb [hunk 1]
```diff
@@ -99,14 +99,18 @@ def CheckAudioQuality(input)
...
   CheckAudioQuality(fileinput)
   MediaConchScan(fileinput)
   $write_to_csv << $file_results
   $file_results = Array.new
 end
 
-CSV.open(File.expand_path("~/Desktop/audioqc-out.csv"), 'wb') do |csv|
+timestamp = Time.now.strftime('%Y-%m-%d_%H-%M-%S')
+CSV.open(File.expand_path("~/Desktop/audioqc-out_#{timestamp}.csv"), 'wb') do |csv|
+  headers = ['Filename','Levels Warnings','Phase Warnings','MediaConch Policy Compliance']
```
Comment (privatezero) [inline audioqc.rb]: Hey! Looks like we were working on this simultaneously! I added an extra column that would go between `'Levels Warnings'` and `'Phase Warnings'` that gives the actual amount of audio frames that exceeded the warning level, so I guess a header needs to be added for that. Something along the lines of `'High Level Frame Number'`

**Comment by privatezero** [comment]
Hi! Haha I was literally just messing with this to make it a little more useful/a little less terrible code! (Still a bunch of bad code in there that I need to redo now that I am better at Ruby lol). 

Apart from needing that one extra column, looks good!

Today I was playing around with the levels - changed it to -2 dB to trigger a warning and added an actual count of high level frames to give a little more context.

I have it pointed at a large oral history collection rn to see if the output seems useful - would love to get input on what kinds of info you are trying to QC off of!

**Comment by mcampos-quinn** [comment]
OK! I was trying to simplify a couple of other things while I add the extra column, but I have to pick up the kid from daycare I'll get to it tomorrow! I am hoping to learn more Ruby (also my kid's name....) so this is kind of a fun exercise for me.
My immediate need is a bunch of digitized audio cassettes, which this seems like it might be really useful for. I want to play with levels and phase error warnings, which seem like they will be more informational than pointing to re-work from our vendor. I want to see if I can incorporate this kind of report into the db records for the digital objects too, maybe....

**Comment by privatezero** [comment]
Cool! I'll hold off on any more messing around until we get this merged. Haha there is definitely a bunch that could be simplified - in looking at it today I noticed I did some things in a VERY circuitous (aka sloppy :-P) manner. Was good practice though I suppose!

I'm kind of excited to pick this up again and make it into something actually useful vs just a dorking around lazy proof of concept, so def welcome input and PRs!

I'm particularly curious to try to dial in a useful level to throw a phase warning on that would work in the context of reformatted audio - I kind of arbitrarily picked this setting. I see that @dericed is alerting for anything less than `-0.18` https://github.com/mediamicroservices/mm/blob/master/mmfunctions#L1306

I haven't gotten around to actually cranking a tape in and out of azimuth to pin down some numbers though!

**Comment by kieranjol** [comment]
I'm not great with audio so it would be great to see how ye end up using
this. Also hi to Ruby!

**Comment by mcampos-quinn** [comment]
Ha hm, I'm also here to learn from Andrew....  Hopefully this approach can get us some helpful reports? We'll see!

File: audioqc.rb [hunk 2]
```diff
@@ -49,8 +49,10 @@ def MediaConchScan(input)
     $policyfile.write(mcpolicy)
     $policyfile.rewind
   end
-  command = 'mediaconch --Policy=' + $policyfile.path + ' ' + '"' + input + '"'
-  mcoutcome = `#{command}`.tr('--','').tr(' ','')
+  policypath = $policyfile.path
+  command = "mediaconch --Policy=#{policypath} '#{input}'"
```
Comment (privatezero) [inline audioqc.rb:53]: Pretty sure the single quotes here around `#{input}` will bork this for working with windows file paths, but gonna go ahead and merge and then tweak it along with some of the other paths!

**Comment by privatezero** [comment]
Stoked to have y'all up in here and to have some conversation! I'll do a bit of refactoring of this today.

My test output from last night (pre-merging Michael's PR) gives me hope this could be useful - with some honing. Definitely needed the context for the amount of high level frames to rule out a TON of false positives. (In this oral history collection there would often be one or two instances of someone hitting the mic etc, which would skew the high peak data). I wonder if it should crunch out two outputs - one more human readable and one more machine parsable?

**Comment by privatezero** [comment]
thanks for the PR @mcampos-quinn!

File: audioqc.rb (hunk 2 above)
Comment (mcampos-quinn) [reply audioqc.rb:53]: d'oh, windows... ok thanks!

Comment (privatezero) [reply audioqc.rb:53]: hahaha I mean, the current way I had it of dumping to ~/Desktop makes it all moot for the moment!

Comment (mcampos-quinn) [reply audioqc.rb:53]: lol... shhh....

**Pull request merged**